cms/djangoapps/contentstore/utils.py

```diff
@@ -163,24 +163,24 @@ def is_currently_visible_to_students(xblock):
     return True
 
 
-def has_children_visible_to_specific_content_groups(xblock):
+def has_children_visible_to_specific_partition_groups(xblock):
```

Comment:
Nit: simplify this methods conditional

`if xblock.has_children:
    for child in xblock.get_children():
        if is_visible_to_specific_content_groups(child):
            return True

return False`

depending on what get_children returns when there are no children available it can probably be simplified more.